spring-boot/admin/src/main/resources/templates/admin/fragments/gateway_health_monitor.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+
+
+<body>
+<div th:fragment="gateway-health-monitor">
+    <div class="x_panel">
+        <div class="x_title">
+            <h2>
+                Gateway Health Monitor
+                <a data-toggle="modal" data-target=".help-gateway-health-monitor">
+                    <i class="fa fa-question-circle"></i>
+                </a>
+            </h2>
+            <div class="clearfix"></div>
+        </div>
+        <div class="x_content">
+            <div class="demo-container" style="height:300px">
+
+                <div th:switch="${gatewayHealthMonitorProxyData.statusSummary}">
+                                    <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).WARNING}">
```

Comment:
It seems like this `span` element is only used for logic, it is not required in the HTML, so maybe use `th:block` instead?

Reply:
Done